.eslintrc.js

```diff
@@ -65,6 +65,27 @@ module.exports = {
       'jsdoc/require-jsdoc': OFF,
       'jsdoc/require-description': OFF
     }
+  }, {
+    files: ['**/dev/**', '**/scripts/**'],
+    rules: {
+      'jsdoc/require-jsdoc': OFF,
+      'jsdoc/require-description': OFF
+    }
+  }, {
+    files: ['packages/@react-aria/interactions/**/*.ts', 'packages/@react-aria/interactions/**/*.tsx'],
+    rules: {
+      'no-restricted-globals': [
+        WARN,
```

Comment:
Choosing to warn here instead of error and only applying to @react-aria/interactions for now, will only switch to error once we have rolled out all the fixes.